prdoc/pr_3659.prdoc

```diff
@@ -0,0 +1,12 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Unity Balance Conversion for Different IDs of Native Asset 
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Introduce types to define 1:1 balance conversion for different relative asset ids/locations 
```

Comment:
What is the purpose of this change? Should probably mention here to the users what has now changed in how the system is used.

Reply:
not sure why you mean, this does not changes anything for users, only introduces few new types. 
it does changes configuration for westend and rococo chains, but it changes it a way it is expected to work